designer/server/src/main/scala/pl/touk/nussknacker/ui/api/UsageStatisticsReportsSettings.scala

```diff
@@ -0,0 +1,26 @@
+package pl.touk.nussknacker.ui.api
+
+import io.circe.generic.JsonCodec
+
+import scala.util.Random
+import pl.touk.nussknacker.engine.version.BuildInfo
+
+import java.net.URLEncoder
+
+private object FingerprintUtils {
+  lazy val random = s"nu-${Random.alphanumeric.take(10).mkString}"
+}
+
+case class UsageStatisticsReportsConfig(enabled: Boolean, fingerprint: String = FingerprintUtils.random)
```

Comment:
One more thing - IMO we should already add fingerprint let's say "development" for instances run from idea / runServer - it looks like env varaible would be nice for that

Reply:
good point, I hope, I detected all places where we shoud override fingerprint:
 * runSever.sh
 * Run configuration in intellij
 * npm run backend:docker script
 
 do you have any idea if thats all ?